### ISSUE TITLE
Contao 4

### ISSUE THREAD

**Issue by nrunde** [47gn/contao_cleverreach_sync#7]
Ist eine Version für Contao 4.4.x bereits vorhanden?

**Comment by sgn-dev** [comment]
nein, gibt es noch nicht. Wird es wahrscheinlich auch nicht geben, Cleverreach hat mittlerweile eine andere API die wir wenn dann für Contao 4.x umsetzen. Aber auch da kann ich leider noch keine Angabe über eine Realisierung machen.

**Comment by Webstylerin** [comment]
Hallo, gibt es irgendwelche Neuigkeiten bzgl. Contao 4? Oder Alternativen zur Integrration von CleverReach?

**Comment by fritzmg** [comment]
Ein Kunde von uns möchte evt. in Zukunft CleverReach verwenden - wir würden dann auf jeden Fall eine mit der neuen API & Contao 4 kompatible Extension entwickeln.

**Comment by sgn-dev** [comment]
@fritzmg - so schaut es bei uns auch aus. Wir hatten schon zwei Anfragen, kam aber beide male nicht zustande. Wenn auf Contao 4 dann ganz sicher mit der neuen API von Cleverreach :)

**Comment by fritzmg** [comment]
@47gn wenn es bei uns etwas zuerst wird, sage ich Bescheid 😁

**Comment by sgn-dev** [comment]
@fritzmg Na klar, ich würde mich auch melden, sollte ein Auftrag hereinflattern :)
